ui/packages/catalog-ui-search/src/main/webapp/js/CQLUtils.js

```diff
@@ -215,7 +215,8 @@ function transformCQLToFilter(cqlString) {
 }
 
 function isPointRadiusFilter(filter) {
-    return filter.value && filter.value.value && filter.value.value.indexOf('POINT') >= 0;
+    var filterValue = typeof(filter.value) === 'object' ? filter.value.value : filter.value;
```

Comment:
✏️ I think we should create a function that returns `filter.value.value` with a descriptive name to increase the readability of this code. I know it's a javascript thing but `.value.value` is not an easy ready in my opinion